blog/_posts/2017-09-12-Final_blog_post.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Hi, thanks for sending your 2nd blog post PR. Please check out the comments below.